include/rtc/rtc.h

```diff
@@ -77,6 +77,28 @@ typedef enum { // Don't change, it must match plog severity
 	RTC_LOG_VERBOSE = 6
 } rtcLogLevel;
 
+#if RTC_ENABLE_MEDIA
+
+typedef enum {
+    // video
+    RTC_H264,
+    RTC_VP8,
+    RTC_VP9,
+
+    // audio
+    RTC_OPUS
+} rtcCodec;
+
+typedef enum {
+    RTC_SENDONLY,
+    RTC_RECVONLY,
+    RTC_SENDRECV,
+    RTC_INACTIVE,
+    RTC_UNKNOWN
```

Comment:
You could put `UNKNOWN` as the first value so it'll map to zero.

Reply:
Done